packages/loopring_v2/contracts/helper/ParticipationHelper.sol

```diff
@@ -109,13 +117,19 @@ library ParticipationHelper {
         }
 
         if ((p.fillAmountS - p.feeAmountS) >= prevP.fillAmountB) {
-            // The miner (or in a P2P case, the taker) gets the margin
-            p.splitS = (p.fillAmountS - p.feeAmountS) - prevP.fillAmountB;
+            // NOTICE: this line commented as order recipient should receive the margin
+            // p.splitS = (p.fillAmountS - p.feeAmountS) - prevP.fillAmountB;
+
+            // Order recipient receives margin
+            p.splitS = 0;
```

Comment:
No splitS trickery needed to store the correct trading history data for buy orders?

Reply:
Pushing that soon.

Reply:
Moved code from another codebase, forgot to pull most recent from that code base before moving it over